packages/account/src/Sections/Profile/PersonalDetails/verify-button.tsx

```diff
@@ -56,6 +56,7 @@ export const VerifyButton = observer(
 
         const redirectToPhoneVerification = (e: React.MouseEvent<HTMLElement>) => {
             e.preventDefault();
+            localStorage.setItem('routes_from_notification_to_pnv', routes.personal_details);
```

Comment:
Can this be added in ui_store rather than using localstorage?

Reply:
@likhith-deriv if user's refresh half way in PNV flow, they will lost the routes if they are coming from notification, that is why we put it in local storage 🙏